lib/chitter.rb

```diff
@@ -0,0 +1,44 @@
+require 'pg'
+
+class Chitter
```

Comment:
This class seems to have a few responsibilities - it's letting you post and view peeps, but it's also handling signup/users. Could those be split into separate classes?